python/lsst/jointcal/jointcal.py

```diff
@@ -172,6 +172,16 @@ class JointcalConfig(pexConfig.Config):
         doc="Write initial/final fit files containing the contributions to chi2.",
         default=False
     )
+    metricOutputFile = pexConfig.Field(
+        dtype=str,
+        doc="Name of the file to write metric measurements to.",
```

Comment:
What directory does this file get written to? Shouldn't it be handled by the butler, or be relative to the butler root or something?

Reply:
I don't know how to make the butler call `job.write()`.  Do you have any suggestions?

Reply:
Further, the system running `jointcal` needs to know where the file goes so that it can be posted to SQuaSH.  It does get written to `cwd`, but you can specify an absolute path.

Reply:
I'm trying to reduce the number of odd files jointcal writes to `cwd` (see e.g. the new `config.writeChi2ContributionFiles`, which defaults to `False`). Having each task write its `Job` output to `cwd` seems like not a viable solution.

Reply:
Oh, in the long term, when we get proper support of new butler datasets it would go away.  I just don't know how to do it with Butler Gen2.

Reply:
In the short term, I can get the path template from the butler and use that to write the file.  Does that work?

Reply:
I think that's a good start. Actually, I wonder if `obs_base` shouldn't gain a `Job` dataset?

Reply:
It has to get added to obs_base, unless we want to add it to all the obs packages.

Reply:
That's my point: put it in obs_base, and then every obs package can use it.

Reply:
Done.